src/render.js

```diff
@@ -41,7 +41,11 @@ export function render(vnode, parentDom, replaceNode) {
 		vnode,
 		oldVNode || EMPTY_OBJ,
 		EMPTY_OBJ,
-		parentDom.ownerSVGElement !== undefined,
+		parentDom.namespaceURI == 'http://www.w3.org/2000/svg'
```

Comment:
Note for myself for tomorrow: double check we can't just pass the namespace through here, the enum might be unnecessary? Could maybe merge the `createElement` and `createElementNS` calls then (assuming the latter doesn't perform worse for HTML elements)?